.gitignore

```diff
@@ -4,8 +4,10 @@ dist
 release
 main.js
 main.js.map
+main.map
 
 *.log
 .DS_Store
 
 Certificates.p12
+.cache/
```

Comment:
This is the `.cache` folder that Parcel uses to cache modules for faster builds.